WikiWatershed/model-my-watershed
# Feature/lol/ie svg issues  2

Connects #914 

This fixes IE svg pattern rendering issues. Each commit fixes one pattern. Mostly the issue was that the majority of the pattern was outside the visible "workspace" of the svg. Other browsers dealt with the appropriately but IE could not recenter on the pattern. 

### To Test
 * Open IE
 * Draw each type of land cover and conservation practice.
 * Ensure all patterns are visible.
 * Also make sure they appear in the modification selection popup.

![screenshot from 2015-10-12 15 51 59](https://cloud.githubusercontent.com/assets/903219/10437992/cd038fd0-70fe-11e5-9022-9dd024a70b65.png)

![screenshot from 2015-10-12 16 26 49](https://cloud.githubusercontent.com/assets/903219/10437993/d52d0420-70fe-11e5-9750-a1ac9c9033f7.png)
